controller/src/main/java/io/pravega/controller/server/rpc/grpc/GRPCServerConfig.java

```diff
@@ -35,4 +35,39 @@
      * @return The RPC port which has to be registered with the cluster used for external access.
      */
     Optional<Integer> getPublishedRPCPort();
+
+    /**
+     * Fetches the settings which indicates whether authorization is enabled.
+     *
+     * @return Whether this deployment has auth enabled.
+     */
+    boolean isAuthorizationEnabled();
+
+    /**
+     * Fetches the location of passwd file for the users.
```

Comment:
I see "passwd" used everywhere in this PR. Does this have a special meaning or is it just a shorthand for "password". If the latter, I really want to see "password" instead (no point in dropping 2 letters ....)